docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx

```diff
@@ -0,0 +1,157 @@
+
+### Support
+
+If you need help with this integration, contact Dagster Cloud support.
+
+---
+
+## Integration walkthrough
+
+### In PagerDuty
+
+1. From the **Configuration** menu, click **Services > Service Directory**.
+
+2. There are two ways to add an integration to a service:
+
+   - **If adding your integration to an existing service**: Click the **name** of the service you want to add the integration to. Then, select the **Integrations** tab and click the **New Integration** button.
+
+   - **If creating a new service for your integration**: Refer to PagerDuty's [Configuring services and integrations documentation](https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations) and follow the steps outlined in the [Create a new service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting **Dagster Cloud** as the **Integration Type** in step 4. Continue with the [**In Dagster Cloud**](#in-dagster-cloud) section (below) once finished.
+
+3. Enter an **Integration Name** in the format `monitoring-tool-service-name` (e.g. `dagster-cloud-run-failures`) and select **Dagster Cloud** from the **Integration Type** menu.
+
+4. Click the **Add Integration** button to save your new integration. You will be redirected to the **Integrations** tab for your service.
+
+5. An **Integration Key** will be generated on this screen. Keep this key saved in a safe place, as it will be used when you configure the integration with **Dagster Cloud** in the next section.
+
+   ![](https://pdpartner.s3.amazonaws.com/ig-template-copy-integration-key.png)
+
+### In Dagster Cloud
+
+Creating an alert policy can be done using the Dagster Cloud UI or in code.
+
+<TabGroup>
+<TabItem name="In the UI">
+
+### In the UI
+
+1. Sign into your Dagster Cloud account.
+
+2. Create and configure an alert policy as per the [alert policy documentation](/dagster-cloud/managing-deployments/alerts#creating-alert-policies).
+
+3. In the **Notification Service** section of the **Create alert policy** window:
+
+   1. Select **PagerDuty**.
+   2. In the field that displays, paste the PagerDuty integration key you created in the previous section.
+
+4. Click **Save policy**.
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+### dagster-cloud CLI
+
+To define a PagerDuty alert policy in code, use the `notification_service.pagerduty.integration_key` key to provide the integration key. For example:
+
+```yaml file=/dagster_cloud/alerts/pagerduty_policy.yaml
+# alert_policies.yaml
+
+alert_policies:
+  - name: "pagerduty-alert-policy"
+    description: "An alert policy to create a PagerDuty incident upon job failure."
+    tags:
+      - key: "level"
+        value: "critical"
+    event_types:
+      - "AGENT_UNAVAILABLE"
+    notification_service:
+      pagerduty:
+        integration_key: TODO: USE ENVVAR?
```

Comment:
I assume we'd want an env var here - what would this look like in this file?

Reply:
We don't support an env var currently- on my list